src/test/java/seedu/multitasky/testutil/CommandUtil.java

```diff
@@ -173,4 +187,76 @@ public static String getDateDetailsAsInputString(Calendar calendar) {
         return builder.toString();
     }
 
+    // @@author A0126623L
+    /**
+     * Returns an list command string for listing active entries.
+     */
+    public static String getListCommand() {
+        return ListCommand.MESSAGE_ACTIVE_SUCCESS;
+    }
+    // @@ author
+
+    // @@author A0126623L
+    /**
+     * Returns an list command string for listing archive.
+     */
+    public static String getListArchiveCommand() {
+        return ListCommand.MESSAGE_ACTIVE_SUCCESS + " " + CliSyntax.PREFIX_ARCHIVE;
```

Comment:
Ah, mistake. I intended to type ListCommand.COMMAND_WORD. Will make the changes when I'm home later